### ISSUE TITLE
app stops instantly if offer is not there

### ISSUE THREAD

**Issue by alialaayedi** [mdesilva/AmazonFlexUnlimited#112]
Hello,

I Have added my desired warehouse which is an amazon fresh store and I have also tried another packages warehouse, the app stops running after seconds for both becuase there is no offers that car currently available. does anyone know how I can make the app keep surching till the offer gets posted?

Terminal:
***Amazon Flex Unlimited v2*** 

INFO: Starting job search...
0 requests attempted

INFO: Job search cycle ending...
INFO: Accepted 0 offers in 3.738399028778076 seconds

**Comment by ssmusic19** [comment]
You need to edit the retryLimit # (probably should be labeled something else) in config.json

**Comment by alialaayedi** [comment]
yes, I figured it was that after posting this sorry to bother you

**Comment by ssmusic19** [comment]
@alialaayedi No worries at all

**Comment by sack2116** [comment]
what is the best optimal setting for "retryLimit": 10,
  "refreshInterval": 0.2,....
is there a concesus as what to use in order to:
1. not get rate limited all the time
2. run the program and accept offers for as long as possible.

is anyone using any command scripts to kill/start the program once rate limited? something aumated?
